src/constants/main.ts

```diff
@@ -30,6 +30,9 @@ export const POINTS_ACHIEVEMENTS: Record<AchievementId, number> = {
 };
 
 //TODO: replace all match stages with these constants
+
+/**** MATCH ****/
+export const MATCH_HOST = "host" as const;
 export const VOTING = "voting" as const;
 export const RESULTS = "results" as const;
 export const CHAT = "chat" as const;
```

Comment:
I don't like this because we lose typesafety since these are now strings instead of being a union of literals. If you define the match stage as a union of literals instead (like it was before) you are guaranteed that the value will never be a random string, but only the ones present in the union